packages/components/src/navigator/types.ts

```diff
@@ -12,16 +12,27 @@ export type NavigatorLocation = NavigateOptions & {
 	isBack?: boolean;
 	path?: string;
 	hasRestoredFocus?: boolean;
+	restoreFocusTo?: string;
 };
 
 export type NavigatorContext = {
 	location: NavigatorLocation;
+	params: any;
+	match?: string;
 	goTo: ( path: string, options?: NavigateOptions ) => void;
 	goBack: () => void;
+	addScreen: ( screen: NavigatorScreen ) => void;
```

Comment:
I thought about "route" initially but I guess our screens are the actual routes.

Reply:
That's a good point — do you think it would make sense to distinguish about `screen` (the React component that gets rendered) and `route` (the abstract object to which a screen is associated) ?

Or we could leave it as-is, and just call it screen everywhere.

Reply:
I have no strong opinion personally. What do you prefer?

Reply:
I think a good middle ground could be to keep `addScreen` and `removeScreen`, but to rename the type from `NavigatorScreen` to simply `Screen` — that would avoid at least partially avoid some confusion